detection-rules/link_download_suspicious_file.yml

```diff
@@ -0,0 +1,51 @@
+name: "Link to auto-download of a suspicious file type (unsolicited)"
+description: |
+  A link in the body of the email downloads a suspicious file type (or embedded file) such as an EXE, LNK, or JS file.
+
+  Recursively explodes auto-downloaded files within archives to detect these file types.
+
+  This technique has been used by known threat actors in the wild.
+type: "rule"
+references:
+  - "https://www.microsoft.com/en-us/security/blog/2021/11/11/html-smuggling-surges-highly-evasive-loader-technique-increasingly-used-in-banking-malware-targeted-attacks/"
+  - "https://twitter.com/MsftSecIntel/status/1418706920152522753"
+  - "https://sandbox.sublimesecurity.com?id=e586c888-4426-41dd-a1be-5a13852075ef"

```

Comment:
this is just pulling up the EML for me, could we include the rule too?

Reply:
i didn't include the rule b/c it can change and that link will get stale. we could tho